src/channel.rs

```diff
@@ -77,10 +79,20 @@ enum Options {
     String(CString),
 }
 
+/// Optimization target for a channel.
+pub enum OptTarget {
+    /// Minimize latency at the cost of throughput.
+    Latency,
+    /// Balance latency and throughput.
+    Blend,
+    /// Maximize throughput at the expense of latency.
+    Throughput,
+}
+
 /// Channel configuration object.
 pub struct ChannelBuilder {
     env: Arc<Environment>,
-    options: HashMap<&'static [u8], Options>,
+    options: HashMap<Cow<'static, [u8]>, Options>,
```

Comment:
why use Cow here?

Reply:
Because both `&[u8]` and `CString` is accepted here.

Reply:
I think we can define three static &u[8] for OptTarget, so we don't need CString here.

Reply:
`CString` is required by `Options`.

Reply:
But I see all your old options don't need it.

Reply:
See L111, L185 and L204.

Reply:
For L111, seem we still can use static u[8] as the key.